packages/frontend/src/analysis/analysis_editor.tsx

```diff
@@ -129,6 +135,7 @@ export function AnalysisDocumentEditor(props: {
                         >
                             <div class="notebook-container">
                                 <h2>Analysis</h2>
+                                <DocumentBreadcrumbs document={props.liveAnalysis} />
```

Comment:
Currently, due to the inconsistent UI idioms around analyses (not having editable titles, and always appearing in a side bar), this looks a little weird. While I very much appreciate that you tested the breadcrumbs in this case too, I think that until we make the UI more consistent (#447), we should show the breadcrumb for *the document being analyzed* in the top-left, that is, [around here](https://github.com/ToposInstitute/CatColab/blob/6371b27c82e9800919d76a31b7e423ae9c2cdf3e/packages/frontend/src/analysis/analysis_editor.tsx#L102). What do you think?

Reply:
I agree strongly. It did look weird and this way the breadcrumbs are in a consistent place. Also the hamburger menu of the left pane is for the analysis, which is consistent with putting the breadcrumbs there.

One of the reasons I put it on the right was so that the spacing at the top of the page would be more consistent, instead I've just added a spacer:

![2025-05-12-133032_hyprshot](https://github.com/user-attachments/assets/7b3a9efc-7ef7-4d64-b9f3-a2b6d4b193eb)